db/migrate/20220802023607_add_user_ref_to_category.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToCategory < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :categories, :user, null: false, foreign_key: true
```

Comment:
* As stated in the [project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=NOTE%3A%20do%20not%20change%20column%20names%20visible%20in%20the%20diagram.%20You%20need%20to%20change%20the%20%22Entity%22%20name%20according%20to%20the%20theme%20you%20have%20chosen%20for%20your%20project%20(please%20note%3A%20%22Transaction%22%20is%20a%20name%20already%20used%20by%20ActiveRecord%2C%20so%20using%20it%20as%20a%20name%20for%20your%20model%20and%20table%20will%20result%20in%20an%20error).) we are asked not to change column names visible in the diagram ( for example instead of `user_id` you should use `Author_id` as the foriegn key)